cmake/ConfigureLinux.cmake

```diff
@@ -0,0 +1,19 @@
+# Copyright 2023 Klepsydra Technologies AG
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set_cxx_standard(11)
+
+set(ADD_OS_COMPILE_OPTIONS
```

Comment:
You can remove this. It's useful only for selecting one option which needs to be disabled in freertos. For linux this is always true (and we don't intend to disable it). The effect of removing this is to reduce one if nesting level in kpsr-core

Reply:
Assuming true in `if(ADD_OS_COMPILE_OPTIONS)` at [kpsr-sdk](https://github.com/klepsydra-technologies/kpsr-sdk/blob/bcd90be99855d7028b350d98992f388628226b76/CMakeLists.txt#L126)?